config/webling-field-mappings.yml

```diff
@@ -555,3 +555,6 @@ mappings:
 - key: legacy
   type: LongTextField
   weblingKey: "(nicht mehr benutzen / plus actuelles)"
+- key: dontUse
+  type: Skip
```

Comment:
Ich nehme an, einfach weglassen ist nicht gegangen.

Reply:
es hätte dann an mehreren Stellen im Code ein `if not "Webling ID / ° ......."` gebraucht, was ich noch unschöner fand...

Reply:
👍